src/Orchard.ContentManagement.Abstractions/ContentItemConverter.cs

```diff
@@ -32,58 +32,67 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
-                {
-                    contentItem.Number = (int)p.Value;
-                }
-                else if (p.Name == nameof(ContentItem.Published))
-                {
-                    contentItem.Published = (bool)p.Value;
-                }
-                else if (p.Name == nameof(ContentItem.PublishedUtc))
-                {
-                    contentItem.PublishedUtc = (DateTimeOffset?)p.Value;
-                }
-                else if (p.Name == nameof(ContentItem.ModifiedUtc))
-                {
-                    contentItem.ModifiedUtc = (DateTimeOffset?)p.Value;
-                }
-                else if (p.Name == nameof(ContentItem.CreatedUtc))
-                {
-                    contentItem.CreatedUtc = (DateTimeOffset?)p.Value;
-                }
-                else if (p.Name == nameof(ContentItem.Author))
+                skip = false;
+
+                if (reader.TokenType == JsonToken.EndObject)
                 {
-                    contentItem.Author = (string)p.Value;
+                    break;
                 }
-                else if (p.Name == nameof(ContentItem.Owner))
+
+                if (reader.TokenType != JsonToken.PropertyName)
                 {
-                    contentItem.Owner = (string)p.Value;
+                    continue;
                 }
-                else
+
+                var propertyName = (string)reader.Value;
+
+                switch (propertyName)
                 {
-                    contentItem.Data.Add(p);
+                    case nameof(ContentItem.Id) : 
+                        contentItem.Id = reader.ReadAsInt32() ?? 0;
+                        break;
+                    case nameof(ContentItem.ContentItemId):
+                        contentItem.ContentItemId = reader.ReadAsString();
+                        break;
+                    case nameof(ContentItem.ContentType):
+                        contentItem.ContentType = reader.ReadAsString();
+                        break;
+                    case nameof(ContentItem.Latest):
+                        contentItem.Latest = reader.ReadAsBoolean() ?? false;
+                        break;
+                    case nameof(ContentItem.Number):
+                        contentItem.Number = reader.ReadAsInt32() ?? 0;
+                        break;
+                    case nameof(ContentItem.Published):
+                        contentItem.Published = reader.ReadAsBoolean() ?? false;
+                        break;
+                    case nameof(ContentItem.PublishedUtc):
+                        contentItem.PublishedUtc = reader.ReadAsDateTimeOffset();
+                        break;
+                    case nameof(ContentItem.ModifiedUtc):
+                        contentItem.ModifiedUtc = reader.ReadAsDateTimeOffset();
+                        break;
+                    case nameof(ContentItem.CreatedUtc):
+                        contentItem.CreatedUtc = reader.ReadAsDateTimeOffset();
```

Comment:
The json date time serialization format contains the timezone, so it will be deserialized with it.